.gitignore

```diff
@@ -1,11 +1,11 @@
-/.bundle/
-/.yardoc
-/_yardoc/
-/coverage/
-/doc/
-/pkg/
-/spec/reports/
-/tmp/
+.bundle
+.yardoc
+_yardoc
+coverage
+doc
+pkg
+tmp
+node_modules
```

Comment:
Why have this ignore both matching directories and files, instead of just directories? The GitHub `.gitignore` templates also have them with the slashes.

This is a nit, but I don't think it needs to change.